code/game/objects/items/tanks/tanks.dm

```diff
@@ -249,7 +263,9 @@
 		return
 
 	//Allow for reactions
-	excited = (excited | air_contents.react(src))
+	var/reacting = air_contents.react(src)
+
+	excited = (excited | reacting)
```

Comment:
?

Reply:
Leftover from when I used the var for another check. Woops.